scripts/liquidation_bot/deploy.ts

```diff
@@ -0,0 +1,60 @@
+import hre from 'hardhat';
+import { DeploymentManager } from '../../plugins/deployment_manager/DeploymentManager';
+import { CometInterface } from '../../build/types';
+import { throwIfMissing } from '../../hardhat.config';
+
+// https://docs.uniswap.org/protocol/reference/deployments
+const UNISWAP_V3_FACTORY_ADDRESS = '0x1F98431c8aD98523631AE4a59f267346ea31F984';
+const WETH9 = '0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2';
+const SWAP_ROUTER = '0xe592427a0aece92de3edee1f18e0157c05861564';
+
+async function main() {
+  const { DEPLOYMENT, RECIPIENT } = process.env;
+  throwIfMissing(DEPLOYMENT, 'missing required env variable: DEPLOYMENT');
+  throwIfMissing(RECIPIENT, 'missing required env variable: RECIPIENT');
+
+  const network = hre.network.name;
+  if (["hardhat", "fuji"].includes(network)) {
+    throw new Error(`Uniswap unavailable on network: ${network}`);
+  }
+
+  const dm = new DeploymentManager(
+    network,
+    DEPLOYMENT,
+    hre,
+    {
+      writeCacheToDisk: true,
+      verificationStrategy: 'eager',
+      debug: true,
+    }
+  );
+  await dm.spider();
+
+  const comet = await dm.contract('comet') as CometInterface;
+  const numAssets = await comet.numAssets();
+  const assets = await Promise.all(Array(numAssets).fill(0).map((_, i) => comet.getAssetInfo(i)));
+
+  const liquidator = await dm.deploy(
+    'liquidator/Liquidator.sol',
+    [
+      RECIPIENT, // _recipient
+      SWAP_ROUTER, // _swapRouter
+      comet.address, // _comet
+      UNISWAP_V3_FACTORY_ADDRESS, // _factory
+      WETH9, // _WETH9
+      0, // _liquidationThreshold,
+      assets.map(a => a.asset), // _assets
```

Comment:
To be clear, we need to deploy a new liquidator if say there's a new asset added to Comet?

Reply:
Great ❓

Reply:
shouldn't be required; the contract will just fallback to the default config:

https://github.com/compound-finance/comet/blob/main/contracts/liquidator/Liquidator.sol#L118